ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java

```diff
@@ -86,20 +86,21 @@ static Properties getProperties(final Arguments arguments) throws IOException {
   }
 
   private static void usage() {
+    String newLine = System.lineSeparator()
```

Comment:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.7.1:site (default-site) on project ksql-parent: Error generating maven-checkstyle-plugin:2.17:checkstyle-aggregate report: Failed during checkstyle configuration: 
Exception was thrown while processing /home/****/workspace/confluentinc-pr_ksql_PR-2651/ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java: 
MismatchedTokenException occurred during the analysis of file /home/****/workspace/confluentinc-pr_ksql_PR-2651/ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java. expecting EOF, found '}' -> [Help 1]
```

Please make sure that this build compiles and add the new help message (copy-paste from output) to the PR description;

Reply:
I've fixed the mistake, and retriggered the CI, but it seems to be timing out. Would expect that to be unrelated to this change though..

Reply:
Weird - I triggered a rebuild and see if that passes. I agree with you that should be unrelated to your change.